pages/instructorIssues/instructorIssues.ejs

```diff
@@ -170,9 +170,23 @@
               <% } else { %>
               <i class="fa fa-check-circle text-success issue-status-icon"></i>
               <% } %>
-              <a class="d-block" href="<%= urlPrefix %>/question/<%= row.question_id %>/?variant_id=<%= row.variant_id %>">
-                <%= row.question_qid %>
-              </a>
+              <div class="d-block">
+                <a href="<%= urlPrefix %>/question/<%= row.question_id %>/?variant_id=<%= row.variant_id %>">
+                  <%= row.question_qid %>
+                </a>
+                <button class="btn btn-xs dropdown-toggle" type="button" id="issue-link-menu-<%= row.issue_id %>"
```

Comment:
I didn't notice this dropdown at first in the UI. It was only because I'd seen the code that I knew where to find it 😄

To make this more visible, how about making the `question_qid` not be a link and adding links afterwards. For example:

**demo/calculation** ([instructor view](#), [student view](#), [assessment details](#))

Reply:
That can work, though I can also make the whole thing a button with the dropdown.

Reply:
It seems that we've got lots of horizontal space available, so I'm inclined to make the links directly clickable without a dropdown?

Reply:
sure, I can do that

Reply:
done.